style.css

```diff
@@ -97,8 +105,37 @@ footer {
   width: 100%;
   bottom: 0;
 }
+#cardFooter {
+  display: flex;
+  flex-direction: column;
+  width: 25vw;
+  height: 30vh;
```

Comment:
you gave the card footer height :30vh but it looks doesnt enough for its content. you may make the div bigger or make the p element margin less.